libs/api/domains/driving-license/src/lib/drivingLicense.service.ts

```diff
@@ -242,10 +242,13 @@ export class DrivingLicenseService {
     nationalId: string,
     type: DrivingLicenseApplicationType,
   ): Promise<ApplicationEligibility> {
-    const assessmentResult = await this.getDrivingAssessmentResult(nationalId)
+    const token = user.authorization.toLowerCase().startsWith('bearer')
+      ? user.authorization.split(' ')[1]
+      : user.authorization // removes the Bearer prefix
+    const assessmentResult = await this.getDrivingAssessmentResult(token)
```

Comment:
Note to self:
Consider trying to standardize these?
Actually it would be better to explicitly call `replace` on 'Bearer' I think